kqueen/models.py

```diff
@@ -38,26 +38,25 @@ class Cluster(Model, metaclass=ModelMeta):
     owner = RelationField(required=True, remote_class_name='User')
 
     def update_state(self):
+        # Check for stale clusters
+        max_age = timedelta(seconds=config.get('PROVISIONER_TIMEOUT'))
```

Comment:
what value do u prefer?

Reply:
imo it can depends on many reasons, need to define to properly

Reply:
it is defined in the config file now, I didn't change that part

Reply:
ok, got it